### PR TITLE
Modify tarchive_series table to include PET DICOMs.

### PR DESCRIPTION
**Pull request by cmadjar**
Adding a Modality field to the tarchive_series in order to differentiate PET and MR images.

### CODE REVIEW

**Comment by driusan** [comment]
This needs to update the default schema too

**Comment by cmadjar** [comment]
Thanks! I added it to the default schema.

File: SQL/0000-00-00-schema.sql [hunk 1]
```diff
@@ -1240,6 +1240,7 @@ CREATE TABLE `tarchive_series` (
   `PhaseEncoding` varchar(255) default NULL,
   `NumberOfFiles` int(11) NOT NULL default '0',
   `SeriesUID` varchar(255) default NULL,
+  `Modality` ENUM ('MR', 'PT') default NULL,
```
Comment (samirdas) [inline SQL/0000-00-00-schema.sql]: This should not have a comma after Modality line

File: SQL/2015-06-13_ModifyTarchiveSeriesTableToIncludePET.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE tarchive_series ADD Modality ENUM ('MR', 'PT');
```
Comment (samirdas) [inline SQL/2015-06-13_ModifyTarchiveSeriesTableToIncludePET.sql]: This should default to NULL, no?

**Pull request merged**